pkg/couchdb/proxy.go

```diff
@@ -114,6 +114,8 @@ func ProxyBulkDocs(db Database, doctype string, req *http.Request) (*httputil.Re
 					var event string
 					if doc.Rev() == "" {
 						event = realtime.EventCreate
+					} else if doc.Get("_deleted") == true {
```

Comment:
You probably need a type conversion here:

```go
} else if deleted, ok := doc.Get("_deleted").(bool); ok && deleted {
```

Reply:
It worked as is.

Reply:
It works, but I'm surprised that gometalinter didn't say anything about that

Reply:
It is working since the result of doc.Get is an interface and interface are comparable.

>  Two interface values are equal if they have identical dynamic types and equal dynamic values or if both have value nil.

Reply:
@nono Why  could the problem be here ?

Reply:
Not a problem. It's just that gometalinter can be very nitpicking sometimes.